README.md

```diff
@@ -63,7 +63,7 @@ First of all, clone the Git repository:
 Then run:
 
     $ mkdir _build && cd _build
-    $ meson ..
+    $ meson ..    # sometimes it's "meson.py"
```

Comment:
I'm not sure I understand. MSYS2 actively eschews any official Windows python on your system: it sets its own PATH. Any meson outside its prefix should be ignored too.

"Sometimes it's "meson.py"" is still accurate :neutral_face: